common/buildcraft/factory/block/BlockTank.java

```diff
@@ -33,7 +33,7 @@
 
 public class BlockTank extends BlockBCTile_Neptune implements ICustomPipeConnection {
     private static final BuildCraftProperty<Boolean> JOINED_BELOW = BuildCraftProperties.JOINED_BELOW;
-    private static final AxisAlignedBB BOUNDING_BOX = new AxisAlignedBB(2.0 / 16.0, 0.0 / 16.0, 2.0 / 16.0, 14.0 / 16.0, 16.0 / 16.0, 14.0 / 16.0);
+    private static final AxisAlignedBB BOUNDING_BOX = new AxisAlignedBB(2 / 16D, 0 / 16D, 2 / 16D, 14 / 16D, 16 / 16D, 14 / 16D);
```

Comment:
Because original line was too long and contains too many dots and zeros, now it's more clean.